packages/components/src/Filter/docs/Filter.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react"
+import { action } from "@storybook/addon-actions"
+import { ComponentMeta, ComponentStory, Story } from "@storybook/react"
+import isChromatic from "chromatic"
+import { StickerSheet } from "../../../../../storybook/components/StickerSheet"
+import { FilterButton, FilterButtonRemovable } from "../../FilterButton"
+import { Filter, FilterContents } from "../../index"
+
+const IS_CHROMATIC = isChromatic()
+
+export default {
+  title: "Components/Filter",
+  component: Filter,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          '`import { Filter, FilterContents, FilterButton, FilterButtonRemovable } from "@kaizen/components"`<br />Use a Filter Button for the renderTrigger.<br />Use FilterContents to wrap the contents within a Filter.',
+      },
+    },
+  },
+} as ComponentMeta<typeof Filter>
```

Comment:
Note: Tried to use `satisfies` instead of `as` and the code snippet died 😂 Something to check for v7